base/bin/migration.mustache

```diff
@@ -2,9 +2,9 @@ const fs = require('fs');
 const mustache = require('mustache');
 
 const now = new Date().toISOString();
-const migrationName = now + '_' + process.argv[2];
+const migrationName = `${now}_${process.argv[2]}`;
```

Comment:
We'll need to update the docker test scripts as well, but that can be done after merge if we miss it.